src/annotator/config/test/settings-test.js

```diff
@@ -521,6 +513,26 @@ describe('annotator.config.settingsFrom', function() {
         jsonSettings: {foo: 'jsonValue'},
         expected: 'jsonValue',
       },
+      {
+        when: 'the defaultValue is null',
```

Comment:
As noted above, I'm not clear on the reason for this. The test that I would expect to see here is that if _no_ `defaultValue` is specified, the function returns null.